articles/paladin_warden/paladin_warden.md

```diff
@@ -0,0 +1,83 @@
+[afromac](https://twitter.com/cormacndaly) - Index Coop  
+
+In the future, the Paladin DAO intends to transfer the control over to the DAO once governance processes are refined and deployed on-chain.
+
+## The Warden protocol
+While the idea of lendable governance power introduces improvements to more trivially designed governance models, the opposite is the case for the vote-escrowed model. In the forefront of a high degree of (or absolute) permissionlesness within DAOs the veModel acts as a protection mechanism against malicious governance behavior. By linking governance power to the duration of the lock period, an effort is made to align the long-term incentives between the DAO and the governance participants. An essential component to maintain the integrity of this mechanism is a whitelist that only allows contracts selected by the DAO to lock in CRV. Accordingly, [it is intended that the locking mechanism cannot be circumvented](https://cryptorisks.substack.com/p/the-vetoken-standard), e.g. through tradable veCRV derivatives.
+
+This is a significant difference from other governance models which was also the reason why separate products labeled “Warden” were developed to specifically cater veCRV and other veTokens.
+
+### Warden boost
+In a nutshell, Warden boost allows the [boosting element of veCRV](https://resources.curve.fi/reward-gauges/boosting-your-crv-rewards) to be rented/leased independently of the underlying voting rights ([see docs for more details](https://doc.paladin.vote/warden/boost-market)). As can be seen from the image below, the lease offer can be configured very precisely.
+
+![Warden veBoost Register Dashboard](wardenBoostUI.png "Warden veBoost Register Dashboard")
+
+Thus, the risks associated with a lease can be limited in advance.
+
+### Warden quest
+The Warden quest product offers a solution that allows protocols to set a budget, duration, the desired amount of votes, and a fixed price per veCRV for gauge vote weights. This distinguishes it from other protocols (e.g. [Votium](https://votium.app/) or Andre’s [bribe.crv.finance](https://bribe.crv.finance/)) where an arbitrary amount can be deposited as an incentive to influence voters. In case the budget remains unused throughout several voting periods, the protocol can withdraw its incentives or adjust the conditions at any time.
+
+For more information on how to create a quest or claim rewards see the [documentation](https://doc.paladin.vote/warden-quest/introduction).
+
+The [audit for Warden quest](https://github.com/spearbit/portfolio/blob/master/pdfs/Paladin-Spearbit-Security-Review.pdf) was carried out by Spearbit whereby all except one issue were resolved. This is due to the fact that Paladin wanted to give its users the possibility to accumulate their earnings over several voting periods. Because the gauge contract does not store information about past votes a high reliance on an accurate Merkle Tree generation persists.
```

Comment:
The last part about a "high reliance on an accurate Merkle Tree generation persists" could be fleshed out more as not all readers may be familiar with the process.